init/Kconfig

```diff
@@ -1917,7 +1917,7 @@ config RUST
 	depends on !MODVERSIONS
 	depends on !GCC_PLUGINS
 	depends on !RANDSTRUCT
-	depends on !DEBUG_INFO_BTF
+	depends on !DEBUG_INFO_BTF || PAHOLE_VERSION >= 124
```

Comment:
Should we follow the other instances of `PAHOLE_HAS_*` and create a `PAHOLE_HAS_LANG_EXCLUDE` or similar?

i.e.

```Kconfig
config PAHOLE_HAS_SPLIT_BTF
	def_bool PAHOLE_VERSION >= 119

config PAHOLE_HAS_BTF_TAG
	def_bool PAHOLE_VERSION >= 123
	depends on CC_IS_CLANG
	help
	  Decide whether pahole emits btf_tag attributes (btf_type_tag and
	  btf_decl_tag) or not. Currently only clang compiler implements
	  these attributes, so make the config depend on CC_IS_CLANG.
```

Reply:
Damn, you are right. Ty for the pointer.

Reply:
No worries!